.circleci/config.yml

```diff
@@ -33,12 +36,9 @@ jobs:
             make test
             bash <(curl -s https://codecov.io/bash)
```

Comment:
Can we use the circleci orb instead of execing a bash script we curl? codecov recently had a security incident where someone hijacked the bash script code.

https://circleci.com/developer/orbs/orb/codecov/codecov